torchvision/datasets/mnist.py

```diff
@@ -4,10 +4,12 @@
 from PIL import Image
 import os
 import os.path
+import gzip
+import lzma
```

Comment:
can you move this import to where it is used? This will make Travis happy, and will make `lzma` not a hard-requirement for torchvision

Reply:
Moving both.